changelog/63095.added

```diff
@@ -0,0 +1 @@
+Restore the previous slack engine and deprecate it, rename replace the slack engine to slack_bolt until deprecation
```

Comment:
We should probably add a separate `.deprecated` changelog for noting the slack engine being depprecated as well as the added